src/mlpack/methods/lsh/lsh_search.hpp

```diff
@@ -159,8 +163,6 @@ class LSHSearch
 
   //! Get the number of projections.
   size_t NumProjections() const { return projections.size(); }
```

Comment:
`cube::size()` actually returns `n_elem`, not `n_slices`, so we should change this to `return projections.n_slices;`.